Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -22,6 +22,20 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
      * @extends eZ.ContentEditViewService
      */
     Y.eZ.ContentCreateViewService = Y.Base.create('contentCreateViewService', Y.eZ.ContentEditViewService, [], {
+        initializer: function () {
+            this.on('*:changeLanguage', this._changeLanguage);
+            this.after('*:languageCodeChange', function (e) {
+                var app = this.get('app'),
+                    activeView = app.get('activeView'),
```

Comment:
that's not super robust if for example we move the language list to a sub view of the ContentEditView. So instead of using the app activeView, you should use the target (`e.target` ie the component which fired the event) of the `changeLanguage` event.